PocketKit/Sources/PocketKit/Tags/MailView.swift

```diff
@@ -0,0 +1,54 @@
+import AVFoundation
+import Foundation
+import MessageUI
+import SwiftUI
+import UIKit
+
+struct MailView: UIViewControllerRepresentable {
+    @Environment(\.presentationMode) var presentation
+    @Binding var result: Result<MFMailComposeResult, Error>?
+    var recipients = [String]()
+    var messageBody = ""
+
+    class Coordinator: NSObject, MFMailComposeViewControllerDelegate {
```

Comment:
wow theres no swift default here? Thanks apple.

Reply:
not that i found. but happy to be told otherwise.